x/blob/types/params.go

```diff
@@ -3,25 +3,26 @@ package types
 import (
 	"fmt"
 
+	"github.com/celestiaorg/celestia-app/pkg/appconsts"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 	"gopkg.in/yaml.v2"
 )
 
 var _ paramtypes.ParamSet = (*Params)(nil)
 
 var (
-	KeyMinSquareSize            = []byte("MinSquareSize")
-	DefaultMinSquareSize uint32 = 1
+	KeyMinSquareSize            = []byte("DefaultMinSquareSize")
```

Comment:
Does this variable really need to add the `Default` prefix? Same for `KeyMaxSquareSize`

Reply:
whoops, leftover bugs from the appconsts PR, I'll refactor